public/custom.css

```diff
@@ -153,6 +144,18 @@ form .hint {
 input[type="radio"].uikit-control-input__input:focus + .uikit-control-input__text:before {
   border-radius: 15px;
 }
+.uikit-btn {
```

Comment:
@marcoalfonso - thoughts?

Reply:
This should restore the styling of "buttons wrongly marked up :)" http://uikit.apps.staging.digital.gov.au/packages/buttons/tests/site/ without changing every link in every article.

Reply:
NO IMPORTANT!